srcpkgs/ueberzug/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ueberzug'
+pkgname=ueberzug
+version=18.1.3
+revision=1
+build_style=python3-module
+hostmakedepends="unzip python3-setuptools"
+makedepends="python3-devel libX11-devel libXext-devel"
+depends="libXext python3-xlib pillow-simd python3-docopt python3-psutil python3-attrs"
+short_desc="Überzug: A command line util which allows drawing images on terminals"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="GPL-3"
```

Comment:
This should be `GPL-3.0-only` or `GPL-3.0-or-later`, following https://spdx.org/licenses/.